card/mod/02_basic_types/set/type/pointer.rb

```diff
@@ -66,9 +70,26 @@ def wrap_item item, args={}
       <div class="add-another-div">#{ link_to 'Add another', '#', :class=>'pointer-item-add'}</div>
     }
   end
-  
   view :list_item do |args|
     %{
+      <li class="pointer-li input-group">
+        <span class="input-group-addon handle">
+          #{ glyphicon 'option-vertical left' }
+          #{ glyphicon 'option-vertical right'}
+        </span>
+        #{ text_field_tag 'pointer_item', args[:pointer_item], :class=>'pointer-item-text form-control' }
+        <span class="input-group-btn">
+          <button class="pointer-item-delete btn btn-default" type="button">
+            #{ glyphicon 'remove-circle'}
+          </button>
+        </span>
+      </li>
+    }
+  end
+
+
+  view :old_list_item do |args|
+    %{
       <li class="pointer-li">
```

Comment:
I had a type/pointer/bootstrap.rb file in the bootstrap module to overwrite this :list_item view but then Type::Skin includes the wrong pointer views. So I have to overwrite it directly in type/pointer.rb. We don't really need the old list item view. The new one works also with the old skins.